File: app/src/main/java/net/sarasarasa/basemvp/mvp/contract/MainContract.java
package net.sarasarasa.basemvp.mvp.contract;

import net.sarasarasa.basemvp.base.IMvpPresenter;
import net.sarasarasa.basemvp.base.IMvpView;

public class MainContract {

    public interface View extends IMvpView {

    }

    public interface Presenter extends IMvpPresenter<View> {

    }

}

File: kotlin/src/main/java/net/sarasarasa/basemvpkotlin/mvp/contract/MainContract.kt
package net.sarasarasa.basemvpkotlin.mvp.contract

import net.sarasarasa.basemvpkotlin.base.IMvpPresenter
import net.sarasarasa.basemvpkotlin.base.IMvpView


class MainContract {

    interface View : IMvpView

    interface Presenter : IMvpPresenter<View>

}
